pkg/actions/webhook.go

```diff
@@ -128,7 +128,7 @@ func (w *Webhook) Run(ctx context.Context, record graveler.HookRecord, buf *byte
 	return nil
 }
 
-func executeAndLogResponse(ctx context.Context, req *http.Request, buf *bytes.Buffer, timeout time.Duration) (n int, err error) {
+func executeAndLogResponse(ctx context.Context, req *http.Request, buf *bytes.Buffer, timeout time.Duration, respMsg interface{}) (n int, err error) {
```

Comment:
I would create a separate function with shared logic with this one, instead of forcing most callers to pass a nil for a value they don't use (at the very least add documentation for this arg).